pom.xml

```diff
@@ -1,98 +1,85 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>org.cmps.tetrahedron</groupId>
     <artifactId>tetrahedron-desktop</artifactId>
     <version>1.0-SNAPSHOT</version>
+    <name>app</name>
 
     <properties>
-        <maven.compiler.source>21</maven.compiler.source>
-        <maven.compiler.target>21</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-
-        <joml.version>1.10.8</joml.version>
-        <lwjgl.version>3.3.4</lwjgl.version>
+        <junit.version>5.9.2</junit.version>
+        <javafx.version>24-ea+5</javafx.version>
+        <javafx.maven.plugin.version>0.0.8</javafx.maven.plugin.version>
     </properties>
 
-    <profiles>
```

Comment:
Is lwjgl version working with all these libraries removed?
both versions of the program must work 
I'm 100% percent sure that they may coexist

Reply:
Yes, you are right, I combined two pom files